debian/rules

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/make -f
+# -*- makefile -*-
+
+ifneq (,$(filter parallel=%,$(DEB_BUILD_OPTIONS)))
+    NUMJOBS = $(patsubst parallel=%,%,$(filter parallel=%,$(DEB_BUILD_OPTIONS)))
+    MAKEFLAGS += -j$(NUMJOBS)
+endif
+
+BOOST_ROOT ?= ~/git/boost
```

Comment:
Debian packaging really needs to be usable from a clean chroot (e.g. via pbuilder).  Boost should be handled via dependencies on the required system boost dev packages.